core/templates/dev/head/profile/Profile.js

```diff
@@ -19,15 +19,21 @@
  */
 
 oppia.controller('Profile', [
-  '$scope', '$http', '$rootScope',
-  function($scope, $http, $rootScope) {
+  '$scope', '$http', '$rootScope', 'oppiaDatetimeFormatter',
+  function($scope, $http, $rootScope, $oppiaDatetimeFormatter) {
```

Comment:
Are there any console errors?

http://webmasters.stackexchange.com/questions/8525/how-to-open-the-javascript-console-in-different-browsers